extensions/amp-analytics/0.1/visibility-manager.js

```diff
@@ -325,10 +357,15 @@ export class VisibilityManager {
       model.setReportReady(createReportPromiseFunc);
     }
 
+    const viewport = Services.viewportForDoc(this.ampdoc);
+
     // Block visibility.
     if (readyPromise) {
       model.setReady(false);
       readyPromise.then(() => {
+        const scrollDepth = viewport.getScrollTop();
+        model.maybeSetInitialScrollDepth(scrollDepth);
```

Comment:
What about if readyPromise doesn't exist. 
One thing to mention is that readyPromise may resolve before we call `maybeSetInitialScrollDepth()`, and the value is not guaranteed to be the initial value. Is that OK?
And can we store the initialScrollDepth under visibility manager instead of visibility model as well?

Reply:
All but last sentence: done.
Last sentence: will discuss with team offline...

Reply:
Re: last sentence, since this should be the scroll depth at the time the creative was loaded, I think this should remain in visibility-model.